### DISCUSSION TITLE
使用opencompass如何自定义数据集，并且自定义Evaluator进行评测指标计算

### DISCUSSION THREAD

**Discussion by pzfok** [open-compass/opencompass#1316] [Q&A]
使用opencompass如何自定义数据集，并且自定义Evaluator进行评测指标计算

**Comment by Liangzi-qyh** [comment]
解决了吗

**Reply by bittersweet1999** [reply]
参见https://github.com/open-compass/opencompass/blob/main/docs/zh_cn/advanced_guides/custom_dataset.md
https://github.com/open-compass/opencompass/blob/main/docs/zh_cn/advanced_guides/new_dataset.md

**Comment by liushz** [comment]
We are considering to refine the document, and if you have any confusion, we welcome your comments to help us refine it.
